edx/edx-app-ios
# User Profile - App shows 'about me' info even if the user selected limited profile

### Description

[LEARNER-433](https://openedx.atlassian.net/browse/LEARNER-433)

App shouldn't show about me info if the user has selected limited profile.

### How to test this PR
1) Login to edX app. 
2) Tap on user profile section on left navigation drawer. 
3) Tap on 'Edit' button. 
4) Select limited profile. 
5) Come back to previous (Edit profile) screen and see there should not be about me info.
